lib/storage/datauri.js

```diff
@@ -27,7 +28,33 @@ async function download(source, file) {
         throw new GenericError(err.message, `${actionName()}_download`);
     }
 }
+/**
+ * Generate Presign Url taking local file path as input
+ * Creates a unqiue name for the file, upload the file to 
+ * storage from the path provided and generates a PreSigned Url
+ * @param {String} Path to local file system, where file is downloaded
```

Comment:
```suggestion
 * @param {String} Path to local file system, where file is available
```
This function does not download the file, so let's not be confusing and write that the file will be available at that path. It should be there when this function is called.